src/main/java/com/microsoft/azure/functions/worker/binding/RpcComplexDataSources.java

```diff
@@ -114,6 +114,8 @@ private HttpRequestMessageImpl(RpcHttpRequestDataSource parentDataSource, Object
         @Override
         public String getMethod() { return this.parentDataSource.httpPayload.getMethod(); }
```

Comment:
We can remove this since we are still in the BETA stage, we accept breaking changes.

Reply:
While it is beta, there are many places where this is being used. One example is the Spring Cloud Function adapter. 

My take is that we should keep the API, add a Builder, and then deprecate those methods when we release 1.0-final

Reply:
It doesn't matter. Because we've already introduced so many breaking changes even like changing the package name.

Reply:
Ok, I will proceed with that cleanup.